src/forms/PayoutDetailsForm/PayoutDetailsForm.js

```diff
@@ -255,7 +255,7 @@ const PayoutDetailsFormComponent = props => (
                   label={streetAddressLabel}
                   placeholder={streetAddressPlaceholder}
                   validate={streetAddressRequired}
-                  onUnmount={() => change('streetAddress', undefined)}
+                  onUnmount={() => form.change('streetAddress', undefined)}
```

Comment:
BTW, what's this for?

Reply:
`Don't use deprecated 'change' but 'form.change' 1a0e9ae`

Dropdown affects visible fields: this clears those fields.